web/src/app/monitor/workbasket/workbasket-planned-date/monitor-workbasket-planned-date.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
If we have no frontend tests, why do we keep the spec files?

Reply:
when you create a component using angular cli it create the standar template. We are keeping these spec files because adding test in the future will be faster.